components/feature/sync/src/test/java/mozilla/components/feature/sync/SyncFeatureTest.kt

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.sync
+
+class SyncFeatureTest
```

Comment:
This Test doesn't really test anything? 😅

Reply:
/me buckles down to try to test this!